docs/product-manuals/operate/deployment/importer-and-archiver.md

```diff
@@ -72,13 +74,14 @@ camunda.operate:
   
 ```
 
-You can further parallelize archiver and(or) importer within one node by using following configuration parameters:
+You can further parallelize archiver and/or importer within one node using the following configuration parameters:
```

Comment:
Uses of and/or should be added to the style guide.

Reply:
Added -- great note!